ArduPlane/Parameters.h

```diff
@@ -124,7 +124,6 @@ class Parameters {
         k_param_rangefinder_landing,
         k_param_land_flap_percent,
         k_param_takeoff_flap_percent,
-        k_param_takeoff_flap_prelaunch,
```

Comment:
btw, the problem with your eeprom was you added this flap_prelaunch right in the middle of this enum and pushed all the other indexes down which corrupted the memory indexes.